manifests/include.pp

```diff
@@ -1,16 +1,17 @@
-define autofs::include(
-  $mapfile = $name,
-  $order   = undef
+#
+define autofs::include (
+  String $mapfile         = $title,
```

Comment:
Why do you prefer title instead name?

Reply:
I was using this as a reference https://puppet.com/docs/puppet/5.5/lang_defined_types.html#title-and-name

But I think name is also good and I can revert it.

Reply:
Ah, nice to know. 

Since mapfile should be unique, $title is better then name.